renovate.json

```diff
@@ -34,27 +35,45 @@
       ],
       "groupName": "etcd dependencies",
       "groupSlug": "etcd-all"
+    }
+  ],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "description": "Update minimum Go version",
+      "managerFilePatterns": [
+        "**/go.mod"
+      ],
+      "matchStrings": [
+        "^go (?<currentValue>\\d+\\.\\d+)\\.0\\b"
+      ],
+      "depNameTemplate": "go",
+      "datasourceTemplate": "golang-version",
+      "extractVersionTemplate": "^(?<version>\\d+\\.\\d+)",
+      "versioningTemplate": "loose"
```

Comment:
Didn't test this at all, as the current codebase doesn't compile with Go 1.23, and I was too lazy to try to rebase this onto an older release in order to test it. We can leave it here, and :crossed_fingers: for the Go 1.25 bump, or I can comment it out until 1.25 is available.